orttraining/orttraining/test/training_api/trainer/trainer.cc

```diff
@@ -333,8 +334,9 @@ int RunTraining(const TestRunnerParameters& params) {
 
       if ((batch_idx + 1) % params.checkpoint_interval == 0) {
         // Save trained weights
-        PathString ckpt_file = ToPathString(
-            params.output_dir + "/ckpt_" + params.model_name + std::to_string(batch_idx));
+        std::basic_ostringstream<PathChar> oss;
+        oss << ToPathString("/ckpt_") << ToPathString(params.model_name) << ToPathString(std::to_string(batch_idx));
```

Comment:
nit: I belive the prefixing / required since we used ConcatPathComponent. Right?

Reply:
nit: And maybe we could use std::ostringstream for oss, afterward, we can use one single ToPathString on it.